core/lib/kv/adapters/upstash.ts

```diff
@@ -2,23 +2,14 @@ import { Redis } from '@upstash/redis';
 
 import { KvAdapter, SetCommandOptions } from '../types';
 
-import { MemoryKvAdapter } from './memory';
-
-const memoryKv = new MemoryKvAdapter();
```

Comment:
How come we removed the memory adapter from the Upstash adapter? This seems unrelated to adding Runtime Cache.

Also, should we have deleted the `./memory.ts` file as well?

Reply:
Memory KV is now handled centrally outside of the individual adapters in the kv/index.ts.